plugins/onex/hooks/scripts/hook-runtime-client.sh

```
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: 2025 OmniNode.ai Inc.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)